pkg/scheduler/scheduler.go

```diff
@@ -109,6 +112,14 @@ func WithProfiles(p ...schedulerapi.KubeSchedulerProfile) Option {
 	}
 }
 
+// WithParallelism sets CPU Threads for Scheduler parallelize package. By default it is 16
+func WithParallelism(threads int32) Option {
+	return func(o *schedulerOptions) {
+		o.parallelism = threads
+		parallelize.Parallelism = int(threads)
```

Comment:
If you are already going to set `parallelize.Parallelism`, there is no need to store `o.parallelism`.